src/silent-blocks/silent-blocks.controller.ts

```diff
@@ -0,0 +1,40 @@
+import { Controller, Get, Param, Res } from '@nestjs/common';
+import { Response } from 'express';
+import { SilentBlocksService } from '@/silent-blocks/silent-blocks.service';
+
+@Controller('silent-block')
+export class SilentBlocksController {
+    constructor(private readonly silentBlocksService: SilentBlocksService) {}
+
+    @Get('height/:blockHeight')
+    async getSilentBlockByHeight(
+        @Param('blockHeight') blockHeight: number,
+        @Res() res: Response,
+    ) {
+        const buffer = await this.silentBlocksService.getSilentBlockByHeight(
```

Comment:
how about adding this within a try catch. and log responses for 4xx error. like block not found.

Reply:
in that case we'll just return an empty block